lib/simulator-xcode-6.js

```diff
@@ -66,6 +67,17 @@ class SimulatorXcode6 extends EventEmitter {
     this.calendar = new Calendar(this.getDir());
   }
 
+  set isHeadless (newValue) {
+    if (newValue) {
+      log.warn('The headless option has no effect on Simulator below Xcode9');
+    }
+    this._isHeadless = !!newValue;
```

Comment:
Outside of Xcode 9 `isHeadless` should always be false, no matter what is sent in to this, no?